nvi-commons/src/main/java/no/sikt/nva/nvi/common/service/model/Candidate.java

```diff
@@ -911,15 +926,32 @@ public Set<CandidateOperation> getAllowedOperations(
         .collect(Collectors.toUnmodifiableSet());
   }
 
-  private boolean hasUnverifiedCreatorFromOrganization(
+  public Set<CandidateIssue> getIssues() {
+    return issues;
+  }
+
+  private List<String> getUnverifiedCreatorsFromOrganization(
       URI topLevelOrganizationId, OrganizationRetriever organizationRetriever) {
     var unverifiedCreators = publicationDetails.getUnverifiedCreators();
     return unverifiedCreators.stream()
-        .flatMap(contributor -> getTopLevelAffiliations(contributor, organizationRetriever))
-        .map(Organization::id)
-        .anyMatch(organizationId -> organizationId.equals(topLevelOrganizationId));
+        .filter(
+            contributor -> {
+              var topLevelAffiliations =
+                  getTopLevelAffiliations(contributor, organizationRetriever)
+                      .map(Organization::id)
+                      .collect(Collectors.toSet());
+              return topLevelAffiliations.contains(topLevelOrganizationId);
```

Comment:
hmm. own method

Reply:
Couldn't quite do that, but simplified a bit